client/src/redux/sagas/jobPostingSaga.js

```diff
@@ -86,6 +86,14 @@ function* fetchJobPostingApplicants({ payload }) {
   }
 }
 
+function* copyStages({ payload }) {
```

Comment:
Tota, nyt en oo varma ja saa korjata, mut tarviiko näitä copyStages ja clearCopiedStaged sagoja? Jos näissä kerran vain kutsutaan uudestaan noita actioneita? Näissähän ei oo mitään muuta toiminnallisuutta?